oblib/data_model.py

```diff
@@ -986,6 +1005,12 @@ def set(self, concept_name, value, **kwargs):
 
         if "precision" in kwargs:
             precision = kwargs.pop("precision")
+        else:
+            precision = None
+        if "decimals" in kwargs:
```

Comment:
Add `decimals` to docstring Args

Reply:
The docstring updates are in https://github.com/SunSpecOrangeButton/pyoblib/pull/122